tool/teleport/common/teleport.go

```diff
@@ -408,7 +408,8 @@ func Run(options Options) (app *kingpin.Application, executedCommand string, con
 }
 
 // OnStart is the handler for "start" CLI command
-func OnStart(config *service.Config) error {
+func OnStart(clf config.CommandLineFlags, config *service.Config) error {
+	config.Log.Infof("Teleport has been started with the configuration located at %q", clf.ConfigFile)
```

Comment:
While we're here, can we also log Teleport version here? "Teleport version v10.0.1 has been started ..."